Applications/WebPortal/WebApp/library/source/pages/account_page/account_controller.tsx

```diff
@@ -26,6 +27,12 @@ interface Properties {
   /** Determines the layout to use based on the display device. */
   displaySize: DisplaySize;
 
+  /** The authenticated user's account. */
+  authUserAccount: Beam.DirectoryEntry;
```

Comment:
Don't use abbreviations. ```account``` or if you really want to disambiguate ```authenticatedAccount``` is good.

Similarly ```roles``` is perfectly fine as a prop.